Documentation-Requirements.md

```diff
@@ -177,3 +193,28 @@ source file and re-generating the binary format image file.
 Ideally, the format of all image source files is an open standard, non-binary
 one such as SVG. Text formats are highly preferable because you can manipulate
 and compare them with standard tools (e.g. `diff(1)`).
+
+# Names
+
+Some documents need to specify the name of people. Such names should be
+written in backticks. The main reason for this is to keep the spell checker
+happy (since it cannot handle all possible names). However, since backticks
+are rendered in a fixed-width font, this makes the names clearer:
```

Comment:
Lines 199-202 suggested rewrite: 
Occasionally documents need to specify the name of people. Write such names in backticks. The main reason for this is to keep the spell checker happy (since it cannot manage all possible names). However, since backticks render in a fixed-width font, this makes the names clearer: